controllers/istio/controller.go

```diff
@@ -29,6 +31,16 @@ const (
 	probeTypeFound
 	probeTypeNotFound
 	probeUnknown
+
+	// possible events
+	FailedRemoveIstioConfigurationForServiceEntryEvent   = "FailedRemoveIstioConfigurationForServiceEntry"
+	RemoveIstioConfigurationForServiceEntryEvent         = "RemoveIstioConfigurationForServiceEntry"
+	FailedRemoveIstioConfigurationForVirtualServiceEvent = "FailedRemoveIstioConfigurationForVirtualService"
+	RemoveIstioConfigurationForVirtualServiceEvent       = "RemoveIstioConfigurationForVirtualService"
+	FailedCreateIstioConfigurationForVirtualServiceEvent = "FailedCreateIstioConfigurationForVirtualService"
+	CreateIstioConfigurationForVirtualServiceEvent       = "CreateIstioConfigurationForVirtualService"
+	FailedCreateIstioConfigurationForServiceEntryEvent   = "FailedCreateIstioConfigurationForServiceEntry"
+	CreateIstioConfigurationForServiceEntryEvent         = "CreateIstioConfigurationForServiceEntry"
```

Comment:
please also double check for all events if they really have to be exported (capitalized).